### ISSUE TITLE
How to test the white-box accuracy

### ISSUE THREAD

**Issue by PingLi00** [YisenWang/MART#2]
hi Yisen Wang, 
When i test the accuracy under white-box attack , I find that the testset  accuracy on MART loss or cross entropy loss is different , So what loss you use when test the white-box accuracy in an article.

**Comment by YisenWang** [comment]
Generally, the test accuracy of FGSM/PGD is based on CE loss.

**Comment by zjysteven** [comment]
@leepingQAQ Note that although people typically use CE loss for PGD attacks, it is also reasonable or even suggested to evaluate under the loss used for training, see http://arxiv.org/abs/2002.08347. More importantly, if you do observe a lower accuracy when using the MART loss for the attacks than CE loss, then that should reflect more accurately how robust MART can achieve.
